Sming/Wiring/CStringArray.h

```diff
@@ -0,0 +1,92 @@
+/****

```

Comment:
Can you move CStringArray.* files to SmingCore/Data/ ?

Reply:
OK. Are you happy with the name? Also, is there a convention you need to stick to for #include guards?

Reply:
> Are you happy with the name?

Well since I cannot come up with a better name I guess this one is ok. If needed we can changed it later on. 

> Also, is there a convention you need to stick to for #include guards?

We don't have a strict convention but as far as I saw from your previous PRs you are doing outstanding in naming the include guards.

Reply:
Eclipse creates the guards - but I do have to remember to change them if the file moves!